lib/matplotlib/backends/backend_wx.py

```diff
@@ -198,7 +199,7 @@ def __init__(self, bitmap, dpi):
             alternative="wxagg", addendum="See the Matplotlib usage FAQ for "
             "more info on backends.")
         RendererBase.__init__(self)
-        DEBUG_MSG("__init__()", 1, self)
+        _log.debug("%s - __init__()", self)
```

Comment:
would use %s with `type(self)` or %r. The plain string representation may be verbose (though it probably is not here - didn't check).

Reply:
sure